src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig

```diff
@@ -27,30 +27,32 @@
 {% form_theme handlingForm 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit.html.twig' %}
 
 {% block shipping_preferences_handling %}
-  <div class="col-xl-10">
-    <div id="handling" class="card">
-      <h3 class="card-header">
-        <i class="material-icons">filter_none</i> {{ 'Handling'|trans }}
+  <div class="row justify-content-center">
```

Comment:
Little BC break as the DOM structure changes. Shall we mark it in the PR ?

Reply:
I see actually you have multiple places where you either add or remove
```
<div class="row justify-content-center">
```

So it's multiple small BC breaks ... unless it's bug fixing ?

Reply:
Well, it's bug fixing at it makes the block size smaller with past structure, but you're right it's DOM structure changes and could be BC Breaks but it should be acceptable at you should never select a row or an helper class with JS!

Reply:
> as you should never select a row or an helper class with JS!

I trust you on this one as you're a way better JavaScript developer than I am 😉

Reply:
It's not about being better, you're taking risk by selecting rows or bootstrap helper classes, except if you're forced to!